libs/librrgraph/src/base/rr_graph_view.h

```diff
@@ -164,6 +164,11 @@ class RRGraphView {
     inline short node_layer(RRNodeId node) const {
         return node_storage_.node_layer(node);
     }
+    
+     /** @brief Get the ptc number twist of a routing resource node. This function is inlined for runtime optimization. */
+    inline short node_ptc_twist(RRNodeId node) const{
+        return node_storage_.node_ptc_twist(node);
```

Comment:
Check if not allocated; return 0 if not.

Reply:
Shouldn't this check that the storage is allocated and error out if it isn't?  (Seems important since we don't always allocate it).